ChineseFilter/sections/specific.txt

```diff
@@ -25,6 +25,7 @@ $script,third-party,denyallow=sharecast.ws|bunnycdn.ru|bootstrapcdn.com|cdn.ampp
 !
 !
 !
+vps234.com###top-bannber
```

Comment:
There's another ad in the footer.
I'd prefer: `||res.vps234.com/resources/images/hosting-promotion/`